### ISSUE TITLE
Helicopter pilots can fastrope

### ISSUE THREAD

**Issue by captainblaffer** [KoffeinFlummi/AGM#2087]
Please disable the ability for pilots to fast-rope out of their own helicopter.

I can't think of any reason why a pilot would want to do this.

More importantly, it can be confusing for the pilot to understand the meaning of 'fast rope' in their interaction menu. They might believe it means cutting the ropes or deploying the ropes. (and not jump out of your own chopper and kill everyone around you)

**Comment by MarcBook** [comment]
That's not a bug it's a feature. The co-pilot can fly the helicopter.

**Comment by hopstar117** [comment]
It's not a bug but at the same time I highly doubt a pilot would be fast roping. But i feel that disabling this would be more emersive

**Comment by AndrewKopczynski** [comment]
It'd be a lot more fair to make it a setting rather than purposely disabling a feature for everyone.

**Comment by dixon13** [comment]
Honestly, I don't see this as a big issue and should not be worked on right now. There are bigger fish to fry. IMO a pilot is not going to fast rope and most likely never will so there is no need to waste time on this.

**Comment by kripto202** [comment]
the problem is that the fast rope button and cut ropes button is way too close to each other. If this feature is going to stay, at least move it where it won't be as easy to click and you have to have a copilot to have the option to sling rope.

**Comment by KoffeinFlummi** [comment]
> More importantly, it can be confusing for the pilot to understand the meaning of 'fast rope' in their interaction menu. They might believe it means cutting the ropes or deploying the ropes. (and not jump out of your own chopper and kill everyone around you)

I'm sorry, but if your pilot isn't able to figure out which of the 2 buttons "Fast Rope" and "Cut Ropes" cuts the ropes, and which will start fast roping out of the heli, he has absolutely no business flying a helicopter, or even playing Arma.

> the problem is that the fast rope button and cut ropes button is way too close to each other. If this feature is going to stay, at least move it where it won't be as easy to click and you have to have a copilot to have the option to sling rope.

No. Just no. Eject is also right next to Gear Down in the action menu. And "Detonate" right next to "Place" in the explosives menu. If you miss that big of a button, you also should not be in charge of a helicopter.

> I can't think of any reason why a pilot would want to do this.

It actually can save your life if you clip your main rotor. Did that once while testing.


TL;DR: We will not introduce additional module options (work for us) because people lack the reading comprehension to understand what "Fast Rope" means.
